app/src/main/res/drawable/ic_category_vector.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="960"
+    android:viewportHeight="960">
+    <path
+        android:fillColor="#FFFFFF"
+        android:pathData="M315 434q-18 0-26.5-15.5T289 388l167-267q8-14 25-14t25 14l167 267q9 15 0.5 30.5T647 434H315zm391 446q-74 0-124-50t-50-124q0-74 50-124t124-50q74 0 124 50t50 124q0 74-50 124t-124 50zm-556-25q-13 0-21.5-8.5T120 825V581q0-13 8.5-21.5T150 551h244q13 0 21.5 8.5T424 581v244q0 13-8.5 21.5T394 855H150z" />
+</vector>
```

Comment:
use FFFFFFFF for color, with an extra FF. And minimize new icons with avocado from https://github.com/alexjlockwood/avocado

Reply:
done (it turns out you have to disable studio's "Reformat Code" in VCS -> Commit to not reset avacado's formatting)